src/tbb/dynamic_link.cpp

```diff
@@ -413,7 +413,7 @@ namespace r1 {
         int flags = RTLD_NOW;
         if (local_binding) {
             flags = flags | RTLD_LOCAL;
-#if __linux__ && !__ANDROID__ && !__TBB_USE_SANITIZERS
```

Comment:
is it possible to have glibc on android ? I believe it is not but did not found and statement in android docs ....

Reply:
android = bionic when it comes to C library.

Reply:
[POSIX.1.2017 does not require `RTLD_DEEPBIND` for `dlopen`](https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/nframe.html) so it seems to be glibc extension.